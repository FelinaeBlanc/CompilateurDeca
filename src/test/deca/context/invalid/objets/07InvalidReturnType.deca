class Invalid {
    boolean b = false;

    int getB() {
        return b;
    }
}