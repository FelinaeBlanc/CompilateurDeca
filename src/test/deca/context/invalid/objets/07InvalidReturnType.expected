Type boolean et int non compatible
